pyamplipi/amplipi.py

```diff
@@ -187,6 +190,36 @@ async def load_preset(self, preset_id: int) -> Status:
     async def announce(self, announcement: Announcement) -> Status:
         response = await self._client.post('announce', announcement.json(**json_ser_kwargs))
         return Status.parse_obj(response)
+    
+    async def get_version(self):
+        if self.version is not None:
+            return self.version
+
+        info = await self.get_info()
+        version_raw = info.version
+        version_nums = []
+        tmp = 0
+        for ch in version_raw:
+            if ch.isdigit():
+                tmp = (tmp*10) + int(ch)
+            else:
+                version_nums.append(tmp)
+                tmp = 0
+        version_nums.append(tmp)
+        self.version = tuple(version_nums[:3])
+        return self.version
+
+    # -- play media call
+    async def play_media(self, media: PlayMedia) -> Status:
+        version = await self.get_version()
+        if version < MIN_MEDIA_PLAYER_VERSION:  # TOO OLD, USE ANNOUNCEMENT
+            announce = Announcement(media=media.media,
+                                    source_id=media.source_id)
+            response = await self._client.post('announce', announce.json(**json_ser_kwargs))
+            return Status.parse_obj(response)
```

Comment:
Should we just call announce?
```suggestion
            return awat self.announce(Announcement(media=media.media,
                                    source_id=media.source_id))
```

Reply:
I assume this works